eras/shelley/impl/src/Cardano/Ledger/Shelley/API/Types.hs

```diff
@@ -26,6 +26,11 @@ import Cardano.Ledger.BaseTypes as X (
 
 -- TODO deprecate these?
 
+-- deprecated
+
+-- deprecated
```

Comment:
These look like leftovers? As well as the TODO [a few lines above ](eras/shelley/impl/src/Cardano/Ledger/Shelley/API/Types.hs)

Reply:
Somehow fourmolu moved them here from lines below where `KeyPair` and `KeyPairs` are imported.

Looks like a bug in fourmolu. I'll remove them if there will be other changes I need to make to this PR.

Reply:
Removed